scripts/curl_install_pypi/install.py

```diff
@@ -41,27 +38,71 @@
     pass
 
 AZ_DISPATCH_TEMPLATE = """#!/usr/bin/env bash
-{install_dir}/{bin_dir_name}/python -m azure.cli "$@"
+{install_dir}/bin/python -m azure.cli "$@"
 """
 
-DEFAULT_INSTALL_DIR = os.path.join(os.path.sep, 'usr', 'local', 'az')
-DEFAULT_EXEC_DIR = os.path.join(os.path.sep, 'usr', 'local', 'bin')
+DEFAULT_INSTALL_DIR = os.path.expanduser(os.path.join('~', 'azure-cli'))
+DEFAULT_EXEC_DIR = os.path.expanduser(os.path.join('~', 'bin'))
 VIRTUALENV_VERSION = '15.0.0'
-BIN_DIR_NAME = 'Scripts' if platform.system() == 'Windows' else 'bin'
 EXECUTABLE_NAME = 'az'
 
-DISABLE_PROMPTS = os.environ.get('AZURE_CLI_DISABLE_PROMPTS')
```

Comment:
The curl script is interactive by design so have a few different prompts.
For users who don't want prompts, they should use the pip (or other) install so they can customize the install.

This was left over from when the curl install was the only way to install the CLI (before the public PyPI packages).